### PR TITLE
Issue 1 - As a guest I would like to see a page for the GASC

### PR DESCRIPTION
**Pull request by jaysonbucy**
I am waiting on the `About` content from the swim team. Also, I want to wait until my meeting with Alex on the 28th to set up the Heroku server.

### CODE REVIEW

**Comment by jaysonbucy** [comment]
I guess this also covers issue-2 for now....

File: Gemfile [hunk 1]
```diff
@@ -1,53 +1,35 @@
...
-git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
-  "https://github.com/#{repo_name}.git"
-end
-
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.0.2'
-# Use sqlite3 as the database for Active Record
 gem 'sqlite3'
```
Comment (kylessnell) [inline Gemfile:4]: you want this for your prod DB?

File: Gemfile [hunk 2]
```diff
@@ -1,53 +1,35 @@
...
-# Use Puma as the app server
 gem 'puma', '~> 3.0'
```
Comment (kylessnell) [inline Gemfile:5]: for a later time, but you'll want to use a different server in production

File: Gemfile [hunk 3]
```diff
@@ -1,53 +1,35 @@
...
-# Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
-# Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
-# Use CoffeeScript for .coffee assets and views
 gem 'coffee-rails', '~> 4.2'
```
Comment (kylessnell) [inline Gemfile]: can kill this if you're not going to use coffee script

File: Gemfile [hunk 4]
```diff
@@ -1,53 +1,35 @@
...
-# See https://github.com/rails/execjs#readme for more supported runtimes
-# gem 'therubyracer', platforms: :ruby
-
-# Use jquery as the JavaScript library
 gem 'jquery-rails'
-# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
```
Comment (kylessnell) [inline Gemfile]: can kill this if you're not doing API response work

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -1,15 +0,0 @@
...
- * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
```
Comment (kylessnell) [inline app/assets/stylesheets/application.css:15]: don't you want this?

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+@import "bootstrap";
+
+/*
+colors for site
+dark blue = #000566
+medium blue = #5FBDF5
+light blue = #C4EAFF
+green = #39B348
+white = #FFF
+*/
```
Comment (kylessnell) [inline app/assets/stylesheets/application.scss]: commented code = bad

File: app/assets/stylesheets/application.scss [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+body{
+  color: #000566;
+  background-color: #C4EAFF;
+}
+
+h1{
+  text-align: center;
+  text-transform: uppercase;
+  text-shadow: 1px 1px 0px #FFF;
+}
```
Comment (kylessnell) [inline app/assets/stylesheets/application.scss:13]: you want all `h1`s in your app styled this way?

File: app/assets/stylesheets/application.scss [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+hr.shadow{
+	height: 10px;
+	border: 0;
+	box-shadow: 0 10px 10px -10px #8c8b8b inset;
```
Comment (kylessnell) [inline app/assets/stylesheets/application.scss]: spacing weird here

File: app/assets/stylesheets/welcome.scss [hunk 9]
```diff
@@ -0,0 +1 @@
+
```
Comment (kylessnell) [inline app/assets/stylesheets/welcome.scss]: get rid of this if you don't need it

File: app/helpers/welcome_helper.rb [hunk 10]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
+end
```
Comment (kylessnell) [inline app/helpers/welcome_helper.rb]: do you need this?

File: app/assets/stylesheets/application.scss (hunk 6 above)
Comment (jaysonbucy) [reply app/assets/stylesheets/application.scss]: old man memory, I can remove it I just found a color schema I liked and didn't want to forget it.

File: app/assets/stylesheets/application.scss (hunk 7 above)
Comment (jaysonbucy) [reply app/assets/stylesheets/application.scss:13]: for now

File: app/views/layouts/application.html.erb [hunk 11]
```diff
@@ -1,14 +1,28 @@
...
+      <div class='header-gasc'>
+        <%= image_tag('gasc_header_2015.png', width: 200, alt: 'GASC') %>
+      </div>
+      <div class='header-login'>
+        Sign In will go here eventually
+      </div>
+    </div>
+    <h1>Greater Ada Swim Club</h1>
+    <%= render "control_bar" %>
+    <hr class="shadow">
```
Comment (kylessnell) [inline app/views/layouts/application.html.erb]: don't look like you're using the bootstrap grid system here.

File: spec/helpers/welcome_helper_spec.rb [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the WelcomeHelper. For example:
+#
+# describe WelcomeHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (kylessnell) [inline spec/helpers/welcome_helper_spec.rb]: kill the commented code

File: spec/helpers/welcome_helper_spec.rb [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+RSpec.describe WelcomeHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (kylessnell) [inline spec/helpers/welcome_helper_spec.rb]: need this?

File: spec/views/welcome/index.html.erb_spec.rb [hunk 14]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "welcome/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (kylessnell) [inline spec/views/welcome/index.html.erb_spec.rb]: need this?

File: app/assets/stylesheets/application.css (hunk 5 above)
Comment (jaysonbucy) [reply app/assets/stylesheets/application.css:15]: I don't think so since I changed it to an scss for sass.

Comment (kylessnell) [reply app/assets/stylesheets/application.css:15]: this is really just meant to be a sprockets file that imports all the other stylesheets in the directory. look at the comments in the original file. if you don't have those require statements you have to have everything in your application.scss file, which isn't ideal as far as organization goes.

Comment (jaysonbucy) [reply app/assets/stylesheets/application.css:15]: but does that still work when you change the file from css to scss? I thought by changing it you removed that functionality by default?

Comment (kylessnell) [reply app/assets/stylesheets/application.css:15]: did you read the comments?

Comment (alexstophel) [reply app/assets/stylesheets/application.css:15]: I agree with Jayson's decision here. In order to access mixins and variables from Bootstrap, you need to use `@import` statements. See the README here: https://github.com/twbs/bootstrap-sass#a-ruby-on-rails.

File: Gemfile [hunk 15]
```diff
@@ -1,53 +1,33 @@
...
+  gem 'sqlite3'
+  gem 'listen'
+  gem 'pry-rails'
+end
+
+group :development, :test do
+  gem 'rspec-rails', '~> 3.0'
+  gem 'rails-controller-testing'
+  gem 'shoulda'
+  gem 'pry'
```
Comment (alexstophel) [inline Gemfile]: Do you need both 'pry' and 'pry-rails'? I think you can just change this line to 'pry-rails' and remove the one from the development block.

**Review by alexstophel — APPROVED** [review]
Nice work!

File: Gemfile (hunk 15 above)
Comment (jaysonbucy) [reply Gemfile]: done

**Pull request closed without merging**